docs/concepts/how-clerk-works.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: How Clerk works
+description: Learn about the main concepts and objects that power Clerk's powerful authentication and user management system.
+type: conceptual
+---
+
+# How Clerk works
```

Comment:
It might be good to link to our `How We Roll` blog series here as well: https://clerk.com/blog/how-we-roll-roundup